src/libpriv/rpmostree-core.c

```diff
@@ -1465,18 +1465,6 @@ throw_package_list (GError **error, const char *suffix, GPtrArray *pkgs)
   return FALSE;
 }
 
-static GVariant*
-gv_nevra_from_pkg (DnfPackage *pkg)
-{
-  return g_variant_ref_sink (g_variant_new ("(sstsss)",
-                             dnf_package_get_nevra (pkg),
```

Comment:
Refresh my memory why we have both the composed/decomposed versions of this?

Reply:
For the overrides case, it's really just to make it easier to consume the output of `--json` in tests. Since bandwidth is a consideration here, I think we can just reassemble it client-side. Will fix up!